package.json

```diff
@@ -16,31 +16,28 @@
     "curl",
     "debug"
   ],
-  "author": "delirius325",
+  "author": "Abhishek Rajeshirke <https://github.com/colidarity>",
   "contributors": [
     {
-      "name": "Justin \"jayceekay\"",
-      "url": "https://github.com/jayceekay"
-    },
-    {
-      "name": "Peter \"binggg\" Zhao",
-      "url": "https://github.com/binggg"
+      "name": "Abhishek Rajeshirke",
+      "url": "https://github.com/colidarity"
     }
   ],
   "license": "MIT",
   "bugs": {
     "url": "https://github.com/delirius325/axios-curlirize/issues"
   },
   "homepage": "https://github.com/delirius325/axios-curlirize#readme",
-  "dependencies": {},
-  "devDependencies": {
+  "dependencies": {
     "axios": ">=0.19.0",
```

Comment:
We also don't need `axios` as a dependency. Users of `axios-curlirize` bring their own `axios` instance.